index.bs

```diff
@@ -23,7 +23,7 @@ Logo: https://webmachinelearning.github.io/webmachinelearning-logo.png
 Deadline: 2023-10-01
 Status Text: <p>
   Further implementation experience and user feedback is being gathered for the
-  {{MLCommandEncoder}} interface that proposes to enable more efficient WebGPU
+  MLCommandEncoder interface that proposes to enable more efficient WebGPU
```

Comment:
If it were introduced in #322, we could leave it here, but it's not mentioned there, and MLCommandEncoder has been removed by another PR, so the name hangs in the void now. We could replace it with "an" (interface that...) or say "the/an experimental MLCommandEncoder interface"?

Reply:
@a-sully What do you say to me just deleting this paragraph for now?

Reply:
This paragraph is soon to be updated in #532, so IMHO any discussions about how exactly to describe our WebGPU interop efforts post-MLCommandEncoder should be had on that PR. For the sake of this PR... could we just this as-is? :)

Reply:
Leaving as-is and dropping the `{{}}` to make Bikeshed happy SGTM.

Reply:
WFM